front/app/components/VoteInputs/multiple/AssignMultipleVotesInput/utils.ts

```diff
@@ -18,12 +18,12 @@ export const getPlusButtonDisabledMessage = (
   maxVotesPerIdeaReached: boolean,
   onIdeaPage?: boolean
 ) => {
-  if (!userHasVotesLeft) return messages.maxVotesReached;
   if (basketSubmitted) {
     return onIdeaPage
       ? messages.votesSubmittedIdeaPage
       : messages.votesSubmitted;
   }
+  if (!userHasVotesLeft) return messages.maxVotesReached;
   if (maxVotesPerIdeaReached) return messages.maxVotesPerIdeaReached;
```

Comment:
Basket having been submitted should take precedence.

Reply:
Is it hard to add a small unit test for this? Red-refactor-green style.